nd4j/samediff-import/samediff-import-api/src/main/kotlin/org/nd4j/samediff/frameworkimport/hooks/PreImportHook.kt

```diff
@@ -50,7 +50,6 @@ interface PreImportHook {
         importGraph: ImportGraph<GeneratedMessageV3, GeneratedMessageV3, GeneratedMessageV3, GeneratedMessageV3, GeneratedMessageV3, GeneratedMessageV3, ProtocolMessageEnum>
     ): HookResult {
         // Parameter docs below are from the onnx operator docs:
-        // https://github.com/onnx/onnx/blob/master/docs/Operators.md#slice
```

Comment:
Was that link removed intentionally?

Reply:
Yes the hook doesn't actually have anything to do with the split op. It was just left over.

Reply:
Then you should probably remove the line above it too.